config.php

```diff
@@ -9,16 +9,17 @@
 	'class' => 'humhub\modules\smartVillage\Module',
 	'namespace' => 'humhub\modules\smartVillage',
 	'events' => [
-		[
-			'class' => TopMenu::class,
-			'event' => TopMenu::EVENT_INIT,
-			'callback' => [Events::class, 'onTopMenuInit'],
-		],
-		[
-			'class' => AdminMenu::class,
-			'event' => AdminMenu::EVENT_INIT,
-			'callback' => [Events::class, 'onAdminMenuInit']
-		],
+        //Hide SmartVillage from header and sidebar for users
```

Comment:
ok thanks so far. if we want, we can comment this in again in the future if we want to implement some additional functionalities.